src/status_im/ui/screens/sync_settings/views.cljs

```diff
@@ -0,0 +1,20 @@
+(ns status-im.ui.screens.sync-settings.views
+  (:require-macros [status-im.utils.views :refer [defview letsubs]])
+  (:require [re-frame.core :as re-frame]
+            [status-im.i18n :as i18n]
+            [status-im.ui.components.react :as react]
+            [status-im.ui.components.status-bar.view :as status-bar]
+            [status-im.ui.components.toolbar.view :as toolbar]
+            [status-im.ui.components.list-item.views :as list-item]
+            [status-im.ui.components.list.views :as list]
+            [status-im.ui.components.colors :as colors]
+            [status-im.constants :as constants]
+            [status-im.ui.components.common.common :as components.common]
+            [status-im.ui.components.icons.vector-icons :as icons]))
+
+(defview sync-settings []
```

Comment:
just defn

Reply:
It does not yet for sure.. This view is just a placeholder for now, and I thought there is a really good heuristical chance that it will need subscriptions.

But I understand. Going to remove it, as, whoever is going to implement it, will add it as necessary.

Reply:
done